<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MitkSegmentationTool</class>
 <widget class="QWidget" name="MitkSegmentationTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>276</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="InfoLabel" name="infoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Instructions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ActionWidget" name="frameCreate">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ActionPushButton" name="createMaskPushBtn">
        <property name="text">
         <string>Create new segmentation</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="checkBoxAutoLabels">
        <property name="text">
         <string>Add labels automatically</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignTop">
    <widget class="ActionPushButton" name="newLabelPushBtn">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Add New Label</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CustomQmitkLabelSetWidget" name="labelSetWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>90</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="toolGUIArea" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QmitkToolSelectionBox" name="toolSelectionBox" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionWidget</class>
   <extends>QFrame</extends>
   <header>ActionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InfoLabel</class>
   <extends>QLabel</extends>
   <header>InfoLabel.h</header>
  </customwidget>
  <customwidget>
   <class>ActionPushButton</class>
   <extends>QPushButton</extends>
   <header>ActionPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkToolSelectionBox</class>
   <extends>QWidget</extends>
   <header location="global">QmitkToolSelectionBox.h</header>
  </customwidget>
  <customwidget>
   <class>CustomQmitkLabelSetWidget</class>
   <extends>QWidget</extends>
   <header>CustomQmitkLabelSetWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
